operations/network/management_network/bican_can_to_chn.md

```diff
@@ -0,0 +1,40 @@
+# BICAN CAN to CHN
+
+
+
+### Retrieve SLS data as JSON
+
+1. Obtain a token.
+
+   ```bash
+   ncn-m001# export TOKEN=$(curl -s -k -S -d grant_type=client_credentials -d client_id=admin-client \
```

Comment:
For 1.3+, the command prompts should not be inside the fenced block. In this case, you would change the earlier line to be:

```
1. (`ncn-m001#`) Obtain a token.